content-service/config.json

```diff
@@ -2,5 +2,6 @@
   "api_key": "test",
   "acao_list": [
     "http://localhost:3001"
-  ]
+  ],
+  "short_url": "http://localhost:3002"
```

Comment:
The same way `api_key` is overridden at the moment: there is a file on production that isn't checked in with the config altered.